### PR TITLE
Inferno Campaign - Steadwick's Fall

### PR DESCRIPTION
**Pull request by tomaas-zeman**
en-08
en-30
en-31
en-11
en-03

### CODE REVIEW

File: assets/maps/inferno_devilish_plan.map
Comment (qwrtln) [inline assets/maps/inferno_devilish_plan.map:1]: This is a great idea! Please, create a separate directory for them. Something like `assets/map-files`?

File: campaigns/inferno_steadwicks_fall.tex [hunk 1]
```diff
@@ -0,0 +1,246 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Inferno Campaign -- Dungeons and Devils}{2. Steadwick's Fall}{\images/firewall.png}
```
Comment (qwrtln) [inline campaigns/inferno_steadwicks_fall.tex:2]: Those numbers (`2.` in this case) don't look so good in the ToC. Maybe for campaigns it should be a custom argument?

File: assets/art/fire_shield.png
Comment (qwrtln) [inline assets/art/fire_shield.png:1]: IMO, this file is too low-res for this purpose. Try maybe ripping it from the original in higher resolution?

File: campaigns/inferno_steadwicks_fall.tex [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+
+\textbf{General Kendal's Skill:} Artillery Ability card\footnote{For General Kendal, the Artilery Ability card always resolves the Expert effect and the Ballista War Machine activates every time the Artillery Ability card is drawn, as well as at the beginning of a Combat Round.}
+
+\textbf{Charging Heroes' Factions:} Rampart, Tower, Castle
+
+\textbf{Charging Heroes' Armies:} Neutral Army is one level higher than your Hero level (max level VI)\footnote{See page 35, ``Field Difficulty Level Table'' in the Core Rulebook, for further details on the number of Neutral Units you have to draw for this Neutral Army.}.
+
+\textbf{Charging Heroes' Deck:} 2 × Might card, 2 × Magic card
+
+\textbf{Charging Heroes' Spell Deck:} 2 × Slow Spell card\footnote{All the Charging Heroes' Enemies use the same AI and Spell decks. Reset them after each Combat.}.
```
Comment (qwrtln) [inline campaigns/inferno_steadwicks_fall.tex:70]:
```suggestion
\textbf{Charging Heroes' Spell Deck:} 2 × Slow Spell card\footnote{All the Charging Heroes' Enemies use the same AI and Spell decks. Reset them after every Combat.}.
```

File: campaigns/inferno_steadwicks_fall.tex [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+Take the following Map tiles and set them up as shown in the scenario map layout:
+
+\textbf{2 × Starting Map tile (I)}
+\begin{itemize}
+  \item 1 × Inferno (S6)
+  \item 1 × Castle (S3)
+\end{itemize}
+
+% \vspace*{\fill}\columnbreak
+
```
Comment (qwrtln) [inline campaigns/inferno_steadwicks_fall.tex:85]:
```suggestion
```

File: campaigns/inferno_steadwicks_fall.tex (hunk 2 above)
Comment (qwrtln) [reply campaigns/inferno_steadwicks_fall.tex:70]: BTW, this footnote doesn't star in a new line.

**Comment by tomaas-zeman** [comment]
> BTW, this footnote doesn't star in a new line.

Not sure how to achieve it. There is the `para` option for the footnote package but if I remove it to have each footnote on its own line, it looks like this - basically separate paragraphs

Screenshot 2024-07-01 at 17 42 59

**Review by qwrtln — APPROVED** [review]

**Pull request merged**